source/playgrounds.txt

```diff
@@ -30,12 +30,31 @@ files with the ``.mongodb`` extension as playgrounds.
 
 .. include:: /includes/admonitions/note-playgrounds.rst
 
-|vsce| splits your Playground window to display the results of your 
-Playground in |json| format in the right-side pane labeled ``Playground 
-Results.json``. If you disabled split-view, |vsce| displays the 
-Playground results in |json| format in a new tab to the right of 
-your Playground. You cannot edit the results in the results pane. 
-You can save the results to a file. 
+|vsce| splits your Playground window to display your
+:guilabel:`Playground Result` in the right-side pane. You can perform
+one of two actions on the result depending on the query you ran in
+the Playground.
+
+- You may save *any* :guilabel:`Playground Result` to a file.
+
+- You may edit any |json| document that |vsce| returns in the
+  :guilabel:`Playground Result` from a
+  :doc:`findOne or find query </read-document-playground>`.
+
+  |vsce| returns these results in |json| format with |json| syntax
+  highlighting. Each |json| document has an :guilabel:`Edit Document`
+  link at its top.
+
+  1. Click :guilabel:`Edit Document` at the top of the document you
+     want to edit.
+
+     |vsce| opens this one |json| document in a new window as an
+     editable document.
+
+  2. Make any changes you need.
+
+  3. Click :guilabel:`Save` to save those changes in the database.
```

Comment:
What is `:guilabel:`Save``? Should we mention here that the document saved by the Cmd/Ctrl + S keyboard shortcut?

Reply:
I changed this to the keyboard command.